docs/values.md

```diff
@@ -205,6 +205,12 @@ The `cidr` within `vipNetworkList` is not required for AWS and Azure Cloud.
 The username/password of the Avi controller is specified with this flag. The username/password are base64 encoded by helm and a corresponding secret
 object is used to maintain the same. Editing this field requires a restart (delete/re-create) of the AKO pod.
 
+### avicredentials.authtoken
+
+The generated authtoken from the Avi controller can be specified with this flag as an alternative to password. The authtoken is also base64 encoded
+and maintained by secret object. The token refresh is managed by AKO. In case of token refresh failure, a new token needs to be generated and updated
```

Comment:
The authtoken is also base64 encoded and updated regularly in a Kubernetes Secret object.

Reply:
updated